src/Microsoft.ML.Transforms/WhiteningTransform.cs

```diff
@@ -108,72 +115,95 @@ public bool TryUnparse(StringBuilder sb)
             }
         }
 
-        public sealed class ColInfoEx
+        public sealed class ColInfo
         {
+            public readonly string Input;
+            public readonly string Output;
             public readonly WhiteningKind Kind;
-            public readonly Float Epsilon;
+            public readonly float Epsilon;
             public readonly int MaxRow;
             public readonly bool SaveInv;
             public readonly int PcaNum;
-            public readonly VectorType Type;
 
-            public ColInfoEx(Column item, Arguments args, ColInfo info)
+            /// <summary>
+            /// Describes how the transformer handles one input-output column pair.
+            /// </summary>
+            /// <param name="input">Name of the input column.</param>
+            /// <param name="output">Name of the column resulting from the transformation of <paramref name="input"/>. Null means <paramref name="input"/> is replaced. </param>
+            /// <param name="kind">Whitening kind (PCA/ZCA).</param>
+            /// <param name="eps">Whitening constant, prevents division by zero.</param>
+            /// <param name="maxRows">Maximum number of rows used to train the transform.</param>
+            /// <param name="saveInverse">Whether to save inverse (recovery) matrix.</param>
```

Comment:
>/// <param name="saveInverse">Whether to save inverse (recovery) matrix.</param> [](start = 12, length = 80)

If purpose of this thing is only to tests, don't expose it.
Make it internal readonly property, and allow to set via arguments, but don't expose it in API.